src/OrchardCore/OrchardCore.Users.Abstractions/Services/IExternalLoginUserToRelateFinder.cs

```diff
@@ -0,0 +1,13 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+
+namespace OrchardCore.Users;
+
+public interface IExternalLoginUserToRelateFinder
+{
+    bool CanHandle(ExternalLoginInfo info);
+
+    Task<IUser> GetUserAsync(ExternalLoginInfo info);
+
+    string GetValueThatLinkAccount(ExternalLoginInfo info);
```

Comment:
maybe rename this method to `GetIdentifierKey`

Reply:
fixed in 4726652